graylog2-server/src/main/java/org/graylog2/plugin/Message.java

```diff
@@ -216,6 +217,7 @@ public DateTime getTimestamp() {
 
         obj.put(FIELD_MESSAGE, getMessage());
         obj.put(FIELD_SOURCE, getSource());
+        obj.put(FIELD_STREAMS, getStreamIds());
```

Comment:
Also, I am not sure if we really should use `getStreamIds()` here because it's using stream functions. I guess those create more objects than the simple for loop we had before?